src/components/main-view/main-view.jsx

```diff
@@ -1,79 +1,83 @@
-            genre: [{
-                name: "Comedy",
-                description: "Comedy is a genre of film in which the main emphasis is on humor. These films are designed to make the audience laugh through amusement and most often work by exaggerating characteristics for humorous effect."
-            }],
-            director: [{
-                name: "Lake Bell",
-                bio: "Lake Siegel Bell is an American actress, screenwriter, and director. She wrote and directed the short film Worst Enemy, which debuted at the Sundance Film Festival in 2012, followed by her 2013 feature film directing debut In a World..., in which she also starred.",
-                birth: "1979"
-            }]
-        },
-        {
-            id: 3,
-            title: "Marie Antoinette",
-            description: "The retelling of France's iconic but ill-fated queen, Marie Antoinette. From her betrothal and marriage to Louis XVI at 14 to her reign as queen at 19 and to the end of her reign as queen, and ultimately the fall of Versailles.",
-            image: "https://m.media-amazon.com/images/M/MV5BNDViNjYxNWYtMWRiOS00NmMyLTgwNDMtNTg3MWYwNjAxNWM4XkEyXkFqcGdeQXVyMjUzOTY1NTc@._V1_FMjpg_UX1000_.jpg",
-            genre: [{
-                name: "Drama",
-                description: "In film and television, drama is a category or genre of narrative fiction (or semi-fiction) intended to be more serious than humorous in tone. A primary element in a drama is the occurrence of conflict—emotional, social, or otherwise—and its resolution in the course of the storyline."
-            }],
-            director: [{
-                name: "Sofia Coppola",
-                bio: "Sofia Coppola is an American director and actress. She acted in The Godfather trilogy and Star Wars Episode I: The Phantom Menace. She directed Lost in Translation, Marie Antoinette, The Virgin Suicides, The Beguiled, On the Rocks, Somewhere and The Bling Ring. She is the daughter of Francis Ford Coppola.",
-                birth: "1971"
-            }]
-        },
-    ]);
 
-const [selectedMovie, setSelectedMovie] = useState(null);
+    const storedUser = JSON.parse(localStorage.getItem("user"));
+
+    const storedToken = localStorage.getItem("token");
+
+    const [user, setUser] = useState(storedUser ? storedUser : null);
+
+    const [token, setToken] = useState(storedToken ? storedToken : null);
+
+    const [movies, setMovies] = useState([]);
+
+    const [selectedMovie, setSelectedMovie] = useState(null);
+
+
+useEffect(() => {
+    if (!token) {
+        return;
+    }
+
+    fetch("https://jens-movie-api.herokuapp.com/movies", {
+        headers: { Authorization: `Bearer ${token}` }
+    })
+    .then((response) => response.json())
+    .then((data) => {
+        const movies = data.docs.map((doc) => {
+            return {
+                id: doc.key,
+                title: doc.Title,
+                image: doc.Image,
+                director: doc.Director_Name?.[0]
```

Comment:
Is this value an array or an object

Reply:
I don't know what I'm supposed to get from this comment. There are {} brackets around it, and this is the example that the Exercise provided.

Reply:
Its good to look at the example but your implementation isnt based on the example, its based on your movie_api. 
Take a look at this file https://github.com/jenscotch/movie_api/blob/main/models.js
according this this file
<img width="534" alt="image" src="https://github.com/jenscotch/myFlix-client/assets/13005417/cdfbb896-3f51-48a2-bbcc-33da73546b3f">
 Director is an object so it should be doc.Director

Reply:
Sorry, but it IS doc.Director???? Are you saying I need to get rid of the "_Name?.[0]"?

Reply:
Yes thats what i am saying, based on your code from movie_api repository, line 11 - 14, Director is an object not an array. If you look at the image or the file link i posted earlier you dont have a Director_Name in your code(https://github.com/jenscotch/movie_api/blob/main/models.js).